.github/workflows/callable.build.yml

```
@@ -15,7 +15,7 @@ jobs:
         with:
           distribution: 'corretto'
           java-version: '21'
-      - uses: gradle/wrapper-validation-action@v3
+      - uses: gradle/actions/wrapper-validation-action@v3
```

The suffix also has to go?
```suggestion
      - uses: gradle/actions/wrapper-validation@v3
```
(applies to all changed files)